rubygem-hammer_cli_foreman_tasks/rubygem-hammer_cli_foreman_tasks.spec

```diff
@@ -0,0 +1,62 @@
+%global gem_name hammer_cli_foreman_docker
+%global confdir hammer
+
+Summary: Contains the code for showing of the tasks (results and progress) in the Hammer CLI
```

Comment:
The gem summary is better actually, although it contains a typo: "Foreman CLI plugin for showing tasks information for resoruces [resources!] and users"